packaging/debian/control

```diff
@@ -3,7 +3,7 @@ Maintainer: Google Cloud Team <gc-team@google.com>
 Section: misc
 Priority: optional
 Standards-Version: 3.9.8
-Build-Depends: debhelper (>= 9.20160709), dh-golang (>= 1.1), golang-go, protobuf-compiler
```

Comment:
why are you removing protobuf-compiler from build deps?

Reply:
We're installing `protoc` ourselves, so I didn't think we'd need it. However, with the most recent commit adding the downloaded `protoc` to path before the `apt` version of `protoc`, we could add it back.